robot/basestation/templates/pages/Base.html

```diff
@@ -15,6 +15,8 @@
     <link href="../../static/css/bootstrap-toggle-2.2.2.min.css" rel="stylesheet">
     <script src="../../static/js/jquery-3.3.1.min.js"></script>
     <script src="../../static/js/helpers.js"></script>
+    <script src="../../static/js/p5.min.js"></script>
```

Comment:
You should just import it on the rover page since we don't need it on other pages. I'm assuming this is processing right?

Reply:
yep, I imported it here cause that's where everything is 
I'll change it